cluster/vagrant/provision-master.sh

```diff
@@ -22,11 +22,7 @@ set -o pipefail
 # See: https://github.com/mitchellh/vagrant/issues/2430
 hostnamectl set-hostname ${MASTER_NAME}
 
-if [[ "$(grep 'VERSION_ID' /etc/os-release)" =~ ^VERSION_ID=21 ]]; then
-  # Workaround to vagrant inability to guess interface naming sequence
-  # Tell system to abandon the new naming scheme and use eth* instead
-  rm -f /etc/sysconfig/network-scripts/ifcfg-enp0s3
-
+if [[ "$(grep 'VERSION_ID' /etc/os-release)" =~ ^VERSION_ID=23 ]]; then
```

Comment:
My intention with this conditional was to limit the fixup to Fedora 21.  The fixup appeared unnecessary (and occasionally problematic) for Fedora 22 and 23, at least for libvirt.  Do the opscode boxes for 23 have different configuration from the fedora libvirt image that does not require this block?